js/apps/account-ui/src/i18n.ts

```diff
@@ -12,6 +12,14 @@ const DEFAULT_LOCALE = "en";
 // that we can have a proper type-safe translation function.
 export type TFuncKey = any;
 
+export const keycloakLanguageDetector: LanguageDetectorModule = {
+  type: "languageDetector",
+
+  detect(): string | undefined {
```

Comment:
This type can be inferred from `LanguageDetectorModule`.
```suggestion
  detect() {
```